core/base/src/TROOT.cxx

```diff
@@ -526,8 +526,23 @@ namespace Internal {
       return macroPath;
    }
 
+   // clang-format off
    ////////////////////////////////////////////////////////////////////////////////
    /// Enables the global mutex to make ROOT thread safe/aware.
+   ///
+   /// The following becomes safe:
+   /// - concurrent construction and destruction of TObjects, including the ones registered in ROOT's global lists (e.g. gListOfCleanups, gListOfFunctions and gListOfFiles)
```

Comment:
gListOfCleanups is not the actually name gROOT->GetListOfCleanups(), etc ...

Reply:
fixed